src/lib389/lib389/cli_conf/replication.py

```diff
@@ -1208,24 +1244,52 @@ def create_parser(subparsers):
     repl_set_per_backend_cl.add_argument('--max-entries', help="The maximum number of entries to get in the replication changelog")
     repl_set_per_backend_cl.add_argument('--max-age', help="The maximum age of a replication changelog entry")
     repl_set_per_backend_cl.add_argument('--trim-interval', help="The interval to check if the replication changelog can be trimmed")
+    repl_set_per_backend_cl.add_argument('--encrypt', action='store_true', help="Set the replication changelog to use encryption.  You must export & import the changelog after setting this.")
+    repl_set_per_backend_cl.add_argument('--disable-encrypt', action='store_true', help="Set the replication changelog to not use encryption.  You must export & import the changelog after setting this.")
 
     repl_get_per_backend_cl = repl_subcommands.add_parser('get-changelog', help='Display replication changelog attributes.')
     repl_get_per_backend_cl.set_defaults(func=get_per_backend_cl)
     repl_get_per_backend_cl.add_argument('--suffix', required=True, help='The suffix that uses the changelog')
 
-    repl_dump_cl = repl_subcommands.add_parser('dump-changelog', help='Decode Directory Server replication change log and dump it to an LDIF')
+    repl_dump_cl = repl_subcommands.add_parser('dump-changelog', help='Dump/export the Directory Server replication changelog to an LDIF')
```

Comment:
If it is not too late we have to decide the naming of CL operation: dump/restore or export/import.
I tend to prefer export/import

Reply:
I do like export/import better, but in RHEL 8 we already documented "dump-changelog" and "restore-changelog" from when Simon did the cl-dump port to python.  So I was trying to create the least amount of doc disturbance :-)  We can still change it though since it's not going to land in RHEL 8.